packages/trace-viewer/src/ui/clickPointer.tsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'react';
+
+declare module 'react' {
+  namespace JSX {
+    interface IntrinsicElements {
+      'x-pw-pointer': JSX.IntrinsicElements['div'];
+    }
+  }
+}
+
+export function ClickPointer({ point }: { point: { x: number; y: number } }) {
+  return (
+    <x-pw-pointer
```

Comment:
Why not a regular div instead? This is a part of UI mode dom, so can be a regular element.

Reply:
Happy to change it to a `div`. Could you elaborate on why we use the custom element in the iframe? I imagined that we'd have special treatment for these components, but I cannot find that in our code.

Reply:
We use custom tags to avoid any user css applying to them. This is useful when these elements are embedded into the page itself, for example inside the snapshot iframe or in the codegen overlay.